eth/api_tracer.go

```diff
@@ -719,6 +719,19 @@ func (api *PrivateDebugAPI) TraceTransaction(ctx context.Context, hash common.Ha
 	return api.traceTx(ctx, msg, vmctx, statedb, config)
 }
 
+// Call lets you trace a given eth_call. returns the structured logs created during the execution of EVM
+// if the given transaction was added on top of the provided block and returns them as a JSON object.
+// You can provide -2 as a block number to trace on top of the pending block.
+func (api *PrivateDebugAPI) Call(ctx context.Context, args ethapi.CallArgs, blockNrOrHash rpc.BlockNumberOrHash, config *TraceConfig) (interface{}, error) {
```

Comment:
The call will under the `debug` namespace, maybe we can rename the function to `TraceCall`? So that in the user side is `debug_traceCall`

Reply:
Yeah good idea